firmware/application/ui_navigation.hpp

```diff
@@ -131,17 +131,20 @@ class StatusTray : public View {
     StatusTray(const StatusTray&) = delete;
     StatusTray& operator=(const StatusTray&) = delete;
 
+    void add_button(ImageButton* child);
     void add(Widget* child);
     void update_layout();
     void clear();
     void paint(Painter& painter) override;
+    uint8_t width() { return width_; };
```

Comment:
Should be able to just use the parent_rect’s left() instead of needing to add this. I’m pretty sure parent_rect() is public on Widget.